api-reference/_hidden/dxChartCommonAnnotationConfig/axis.md

```diff
@@ -5,6 +5,6 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the [name](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/valueAxis/name.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/#name') of the [value axis](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/valueAxis '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/') on which the [value](/api-reference/_hidden/dxChartCommonAnnotationConfig/value.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#value') is specified. Useful for a [multi-axis chart](/concepts/05%20Widgets/Chart/20%20Axes/70%20Multi-Axis%20Chart.md '/Documentation/Guide/Widgets/Chart/Axes/Multi-Axis_Chart/').
+Specifies the [name](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/valueAxis/name.md '{basewidgetpath}/Configuration/valueAxis/#name') of the [value axis](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/valueAxis '{basewidgetpath}/Configuration/valueAxis/') on which the [value](/api-reference/_hidden/dxChartCommonAnnotationConfig/value.md '{basewidgetpath}/Configuration/annotations/#value') is specified. Useful for a [multi-axis chart](/concepts/05%20Widgets/Chart/20%20Axes/70%20Multi-Axis%20Chart.md '/Documentation/Guide/Widgets/Chart/Axes/Multi-Axis_Chart/').
```

Comment:
Why is it 'Useful for'?

Reply:
An old fragment of text that I did not intend to rewrite. As for now, I would say that this sentence highlights one of the possible scenarios. I think that I will remove the last sentence

Reply:
Ok